sdk/resources/mgmt/src/main/java/com/azure/management/resources/fluentcore/utils/PagedList.java

```diff
@@ -0,0 +1,364 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.management.resources.fluentcore.utils;
+
+import com.azure.core.http.rest.PagedIterable;
+import com.azure.core.http.rest.PagedResponse;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.ConcurrentModificationException;
+import java.util.Iterator;
+import java.util.List;
+import java.util.ListIterator;
+import java.util.NoSuchElementException;
+import java.util.Objects;
+
+/**
+ * Defines a list response from a paging operation. The pages are
+ * lazy initialized when an instance of this class is iterated.
+ *
+ * @deprecated Use {@link PagedIterable} instead.
+ *
+ * @param <E> the element type.
+ */
+@Deprecated
```

Comment:
Annotate as deprecated.

Reply:
what's the deprecation mean? Should v2 still use PagedList or should not?